fern/openai-realtime.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: OpenAI Realtime
+subtitle: You can use OpenAI's newest speech-to-speech model with your Vapi assistants.
+slug: openai-realtime
+---
+
+OpenAI's Realtime API enables developers to create low-latency, voice-driven interactions within applications by leveraging a speech-to-speech model. This API processes audio input and output in real-time, supporting seamless, natural conversations without intermediate text conversion.
+
+To start using it with your Vapi assistants, select `gpt-4o-realtime-preview-2024-10-01` as your model. 
+- Please note that only OpenAI voices may be selected while using this model. The voice selection will not act as a TTS (text-to-speech) model, but rather as the voice used within the speech-to-speech model.
```

Comment:
nit edits if helpful (basically felt that saying `low-latency, voice-driven interactions within application` felt redundant since that's all of Vapi, and also sounds a bit like an ad for it), so just tried phrasing differently. also added some additional clarification. feel free to use whatever bits are helpful.


> OpenAI’s Realtime API enables developers to use a native speech-to-speech model. Unlike other Vapi configurations which orchestrate a transcriber, model and voice API to simulate speech-to-speech, OpenAI’s Realtime API natively processes audio in and audio out.
> 
> To start using it with your Vapi assistants, select gpt-4o-realtime-preview-2024-10-01 as your model.
> * Please note that only OpenAI voices may be selected while using this model. The voice selection will not act as a TTS (text-to-speech) model, but rather as the voice used within the speech-to-speech model.
> * Also note that we don’t currently support Knowledge Bases with the Realtime API.
> * Lastly, note that our Realtime integration still retains the rest of Vapi's orchestration layer such as the endpointing and interruption models to enable a reliable conversational flow.